homeassistant/components/buienradar/sensor.py

```diff
@@ -4,6 +4,7 @@
 import logging
 
 from buienradar.constants import (
+    __BRCONDITIONS,
```

Comment:
The double underscore indicates it is a private constant and should thus not be used externally.